### PR TITLE
Ignores env file from repo

### PR DESCRIPTION
**Pull request by manuelurenah**
### Trello Ticket Url

- https://trello.com/c/0rIzIigk/227-bug-secret-key-posted-in-clay-starter-repo

### Description

- Adds `.env` to the `.gitignore` to avoid uploading secrets and keys
- Adds a `.env.sample` file to serve as placeholder for `.env` file
- This closes issue #41

### CODE REVIEW

**Comment by manuelurenah** [comment]
[[BUG] Secret key posted in clay-starter repo](https://trello.com/c/0rIzIigk/227-bug-secret-key-posted-in-clay-starter-repo)

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ start-services:
 	docker-compose up -d nginx postgres redis elasticsearch
 
 start-clay:
-	cd app && npm i && npm run start:dev
+	cd app && cp .env.sample .env && npm i && npm run start:dev
```
Comment (PedroAlbR) [inline Makefile]: This will overwrite the `.env` file every time you start the app.

Comment (manuelurenah) [reply Makefile]: Nice catch! Do you think it would be best to add this step in the README?

Comment (PedroAlbR) [reply Makefile]: Yes, that sounds like a good idea

File: README.md [hunk 2]
```diff
@@ -30,9 +31,9 @@ You should be able to navigate to http://localhost/_pages/sample-article.html to
 
 ### Accessing The Edit UI
 
-The edit interface of Clay is a component itself called [`Kiln`](https://github.com/clay/clay-kiln)! To begin editing with the UI you'll need to make sure you've run `make boostrap-user` after replacing the sample user with your own Gmail address.
+The edit interface of Clay is a component itself called [`Kiln`](https://github.com/clay/clay-kiln)! To begin editing with the UI you'll need to make sure you've run `make boostrap-user` after replacing the sample user with your own credentials or using the default one.
```
Comment (james-owen) [inline README.md]: It might be helpful to have an example of a username/provider, just so folks know what sort of thing they'll have to include.

**Pull request merged**